src/components/layout/UserMenu.js

```diff
@@ -0,0 +1,188 @@
+                        id={build(baseId, ids.NAME_LBL)}
+                        variant="caption"
+                        className={classes.typography}
+                    >
+                        {name}
+                    </Typography>
+                </Grid>
+                <Grid item>
+                    <Typography
+                        id={build(baseId, ids.USERNAME_LBL)}
+                        variant="caption"
+                        className={classes.typography}
+                    >
+                        {t("username")}: {id}
+                    </Typography>
+                </Grid>
+                <Grid item>
+                    <Typography
+                        id={build(baseId, ids.EMAIL_LBL)}
+                        variant="caption"
+                        className={classes.typography}
+                    >
+                        {email}
+                    </Typography>
+                </Grid>
+                <Grid item>
+                    <Button
+                        id={build(baseId, ids.MANAGE_ACCOUNT_BTN)}
+                        variant="outlined"
+                        className={classes.button}
+                        onClick={onManageAccountClick}
+                        size="small"
+                    >
+                        {t("manageAccount")}
+                    </Button>
+                </Grid>
+            </Grid>
+            <Divider className={classes.divider} />
+            <Grid
+                container
+                direction="column"
+                justify="center"
+                alignItems="center"
+                spacing={1}
+            >
+                <Grid item>
+                    <Button
+                        id={build(baseId, ids.LOGOUT_BUTTON)}
+                        variant="outlined"
+                        className={classes.button}
+                        onClick={onLogoutClick}
+                        size="small"
+                    >
+                        {t("logout")}
+                    </Button>
+                </Grid>
+            </Grid>
+            <Divider className={classes.divider} />
+            <Grid
+                container
+                direction="row"
+                justify="center"
+                alignItems="center"
+                spacing={1}
+            >
+                <Grid item>
+                    <ExternalLink
+                        id={build(baseId, ids.POLICY_LINK)}
+                        href={constants.CYVERSE_POLICY_URL}
+                    >
+                        <Typography
+                            variant="caption"
+                            className={classes.typography}
+                        >
+                            {t("polices")}
+                        </Typography>
+                    </ExternalLink>
+                </Grid>
+                <Grid item>
+                    <Typography variant="caption">.</Typography>
```

Comment:
I think, if we're trying to replicate Google's user menu, this is actually a • dot (Opt+8 on mac) and not a regular . period.  Switching in the dot should help it look a little more vertically centered

Reply:
Lol. GTK that it is (opt + 8) cos I was searching for a bigger dot than a period in my keyboard 😄